utils/node_local_test.py

```diff
@@ -594,31 +594,30 @@ def start(self):
                              '--undef-value-errors=no']
             self._io_server_dir = tempfile.TemporaryDirectory(prefix='dnt_io_')
 
-            fd = open(os.path.join(self._io_server_dir.name,
-                                   'daos_engine'), 'w')
+            fd = open(join(self._io_server_dir.name, 'daos_engine'), 'w')
             fd.write('#!/bin/sh\n')
             fd.write('export PATH=$REAL_PATH\n')
             fd.write('exec valgrind {} daos_engine "$@"\n'.format(
                 ' '.join(valgrind_args)))
             fd.close()
```

Comment:
Isn't this a case where `with` should be used?
```suggestion
            with open(join(self._io_server_dir.name, 'daos_engine'), 'w') as fd:
                fd.write('#!/bin/sh\n')
                fd.write('export PATH=$REAL_PATH\n')
                fd.write('exec valgrind {} daos_engine "$@"\n'.format(' '.join(valgrind_args)))
```

Reply:
Thanks.